app/tasks/raster_tile_cache_assets/raster_tile_cache_assets.py

```diff
@@ -62,6 +64,15 @@ async def raster_tile_cache_asset(
         ).replace("{tile_id}.tif", "tiles.geojson")
     ]
 
+    # Create a generic calc string when a raster tile cache is being
+    # created from a multi-band raster tile set.
+    band_count = source_asset.creation_options["band_count"]
+    if band_count > 1:
+        bands_string = str(list(string.ascii_uppercase[:band_count])).replace("'", "")
```

Comment:
Maybe put a comment describing what you're doing here for the bands_string, since there's a lot going on on this line

Reply:
Done